content/docs/dvclive/get-started.md

```diff
@@ -116,51 +78,98 @@ for epoch in range(NUM_EPOCHS):
     live.next_step()
 ```
 
-## Outputs
+</tab>
+</toggle>
+
+Check the [ML Frameworks](/doc/dvclive/api-reference/ml-frameworks) page for
+more details and other supported frameworks.
+
+If you prefer to use DVCLive directly, check the overview of the
+[Python API](/doc/dvclive/api-reference).
```

Comment:
Minor/not blocking, but you could drop this now that you added the Python API (which looks good IMO).